File: src/utils/web/config/contentConfig.ts
/**
 * @author sushao
 * @since 0.4.0
 * @version 0.4.0
 * @description 文章的配置
 * */
export interface ContentConfig {
  /**
   * 编码方式
   * */
  encoding: string;
  /**
   * 章节名
   * */
  chapterName: string;
  /**
   * 小说名
   * */
  novelName: string;
  /**
   * 上一章
   * */
  preChapterId: string;
  /**
   * 下一章
   * */
  nextChapterId: string;
  /**
   * 文章内容
   * */
  content: string;
  /**
   * 文章内容切割
   * */
  contentSplit: string | null;
}

File: data/src/store/config/content_config.rs
use serde::{Deserialize, Serialize};
#[derive(Serialize, Deserialize, Debug, Clone)]
pub struct ContentConfig {
    pub(crate) encoding: String,
    #[serde(rename = "chapterName")]
    pub(crate) chapter_name: String,
    #[serde(rename = "novelName")]
    pub(crate) novel_name: String,
    #[serde(rename = "preChapterId")]
    pub(crate) pre_chapter_id: String,
    #[serde(rename = "nextChapterId")]
    pub(crate) next_chapter_id: String,
    pub(crate) content: String,
    #[serde(rename = "contentSplit")]
    pub(crate) content_split: Option<String>,
}
